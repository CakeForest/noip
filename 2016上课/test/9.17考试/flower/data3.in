10 10
2 4 7
1 4 10
1 2 9
1 5 6
2 1 7
2 2 5
2 1 6
1 3 3
1 1 3
1 5 9
