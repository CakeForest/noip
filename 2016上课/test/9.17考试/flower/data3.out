0
4 9
2 3
Can not put any one.
6
0
0
3 5
1 6
7 7
